### PR TITLE
Add build script for Keycloak

### PR DESCRIPTION
**Pull request by Chandranana**
## Checklist

- [X] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [X] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: k/keycloak/keycloak_ubi_8.7.sh [hunk 1]
```diff
@@ -0,0 +1,126 @@
+#!/bin/bash -ex
+# -----------------------------------------------------------------------------
+#
+# Package	: keycloak
+# Version	: v22.0.4
```
Comment (nikhil-kalbande) [inline k/keycloak/keycloak_ubi_8.7.sh:5]: this should be without v?

File: k/keycloak/keycloak_ubi_8.7.sh [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  fi
+
+  PROJECTS="$PROJECTS$SEP$i"
+  SEP=","
+done
+
+if ! mvn test -nsu -B -pl "$PROJECTS" -am; then
+	echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_success_but_test_Fails"
+	exit 0
```
Comment (nikhil-kalbande) [inline k/keycloak/keycloak_ubi_8.7.sh:84]: should be exit 2 for test failure

File: k/keycloak/build_info.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "maintainer": "Naik.Chandranana@ibm.com",
+    "package_name" : "keyclaok",
```
Comment (nikhil-kalbande) [inline k/keycloak/build_info.json:3]: correct typo here

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: k/keycloak/keycloak_ubi_8.7.sh (hunk 1 above)
Comment (Chandranana) [reply k/keycloak/keycloak_ubi_8.7.sh:5]: Done

File: k/keycloak/keycloak_ubi_8.7.sh (hunk 2 above)
Comment (nikhil-kalbande) [reply k/keycloak/keycloak_ubi_8.7.sh]: @Chandranana  This is still not done?

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**